py-polars/polars/internals/expr.py

```diff
@@ -2235,14 +2239,12 @@ def buckets(self, interval: timedelta) -> Expr:
         Examples
         --------
         >>> from datetime import datetime, timedelta
-        >>> import polars as pl
```

Comment:
We use the convention, following numpy, Pandas and others, to assume that `import polars as pl` has been done.